modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/file-size-limit/FileSizeMimetypes.js

```diff
@@ -22,81 +22,110 @@ import uuidv4 from 'uuid/v4';
 
 import '../../css/file_size_mimetypes.scss';
 
+const NumberErrorMessage =
+	Liferay.Language.get('error-colon') +
+	' ' +
+	Liferay.Language.get('please-enter-a-valid-number');
```

Comment:
We can use string templates here 😄

Reply:
Added new commit with this change, thanks!